File: design-patterns/go/factory/transport/car.go
// car.go

package main

import "fmt"

type Car struct {
}

func NewCar() (car *Car) {
	car = &Car{}
	return
}

func (car *Car) Repair() {
	fmt.Println("Car Repair")
}

func (car *Car) Start() {
	fmt.Println("Car started")
}

func (car *Car) Stop() {
	fmt.Println("Car Stopped")
}

File: design-patterns/php/factory/transport/Car.php
<?php
// Car.php

namespace BigBoxCode\DesignPattern\Factory\Transport;

use BigBoxCode\DesignPattern\Factory\Transport\Transport;

class Car implements Transport {
    public function start() {
        echo "Car started\n";
    }

    public function stop() {
        echo "Car Stopped\n";
    }
    
    public function repair() {
        echo "Car Repair\n";
    }
}